discord/iterators.py

```diff
@@ -852,10 +852,10 @@ class ScheduledEventSubscribersIterator(_AsyncIterator[Union["User", "Member"]])
     def __init__(
         self,
         event: ScheduledEvent,
-        limit: int,
+        limit: int | None,
```

Comment:
No default value?

Reply:
> No default value?

it is only used in `subscribers`:
<img width="415" alt="image" src="https://github.com/Pycord-Development/pycord/assets/61660055/4ea95c2b-aa3b-4e30-8b85-896a45774916">
<img width="512" alt="image" src="https://github.com/Pycord-Development/pycord/assets/61660055/07ad765d-e5d1-4552-bf31-05d80161c7a0">